.github/workflows/build-and_deploy_doxygen.yml

```diff
@@ -0,0 +1,30 @@
+name: GitHub Actions Build and Deploy Doxygen
+run-name: ${{ github.actor }} is building and deploying Doxygen
+on: [push]
+jobs:
+  Build-And-Deploy-Doxygen:
```

Comment:
nitpick: I think the standard (or accepted practice) is naming jobs with lowercase letters, i.e. build-and-deploy-doxygen

Reply:
Thank you, resolved both.